src/components/SearchForm/FormFields.tsx

```diff
@@ -176,37 +184,42 @@ export const MetastasisAutocomplete = ({
 }: {
   field: ControllerRenderProps<SearchFormValuesType, 'metastasis'>;
 }): ReactElement => (
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Is there a reason Prettier is disabled for this line? If there is, it should be mentioned with a comment.